modules/interactivity.js

```diff
@@ -0,0 +1,29 @@
+const items = JSON.parse(localStorage.getItem('TodoList')) || [];
+
+const itemCheck = () => {
+  const checkboxes = document.querySelectorAll('.checkbox');
+
+  checkboxes.forEach((checkbox, index) => {
+    checkbox.addEventListener('change', (e) => {
+      if (items[index]) {
+        items[index].completed = e.target.checked;
+        localStorage.setItem('TodoList', JSON.stringify(items));
+      }
+    });
+  });
+};
+
+const clearCompleted = () => {
+  const clearBtn = document.querySelector('.btn-clear');
+
+  clearBtn.addEventListener('click', () => {
+    const newItems = items.filter((item) => item.completed === false);
+    newItems.forEach((todo, idx) => {
+      todo.index = idx + 1;
+    });
+    localStorage.setItem('TodoList', JSON.stringify(newItems));
+    window.location.reload();
+  });
+};
```

Comment:
- [ ] Kindly make sure when the user clicks on the `Clear all completed` button, all checked tasks must be deleted but at this moment your app is deleting everything when this button is clicked. :wink: 

   - ### YOUR APP BEHAVIOUR
    
     ![chrome-capture-2022-10-12 (9)](https://user-images.githubusercontent.com/87197412/201489291-c13d0e80-52c6-4336-93ee-4d8ada29e557.gif)